<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersPage</class>
 <widget class="QWidget" name="UsersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget { background-color: #F8FAFC; }
    font-family: &quot;Segoe UI&quot;;
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="header_layout">
     <item>
      <layout class="QVBoxLayout" name="title_layout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_title">
         <property name="styleSheet">
          <string>font-size: 30px; font-weight: 800; color: #0F172A;</string>
         </property>
         <property name="text">
          <string>User Accounts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_subtitle">
         <property name="styleSheet">
          <string>font-size: 16px; color: #64748B;</string>
         </property>
         <property name="text">
          <string>Manage staff access and roles</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="header_spacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_user">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>45</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
            background-color: #0891B2;
            color: white;
            border-radius: 12px;
            font-weight: bold;
            font-size: 14px;
            padding: 10px;
         }
         QPushButton:hover {
            background-color: #0E7490;
         }
        </string>
       </property>
       <property name="text">
        <string>  Add User</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_list_card">
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QFrame#frame_list_card {
            background-color: white;
            border-radius: 16px;
            border: 1px solid #E2E8F0;
        }
      </string>
     </property>
     <layout class="QVBoxLayout" name="vbox_card">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="list_header">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
            background-color: #F8FAFC;
            border-bottom: 1px solid #E2E8F0;
            border-top-left-radius: 16px;
            border-top-right-radius: 16px;
         </string>
        </property>
        <layout class="QHBoxLayout" name="hbox_lh">
         <property name="leftMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_active_staff">
           <property name="styleSheet">
            <string>color: #334155; font-weight: 800; font-size: 12px; letter-spacing: 1px;</string>
           </property>
           <property name="text">
            <string>ACTIVE STAFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_users">
        <property name="styleSheet">
         <string>background: transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>898</width>
           <height>371</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="layout_users_list">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="vspacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottom_spacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
